lib/mocha.js

```diff
@@ -1229,8 +1229,12 @@ Mocha.prototype._runGlobalFixtures = async function _runGlobalFixtures(
   fixtureFns = [],
   context = {}
 ) {
-  for await (const fixtureFn of fixtureFns) {
-    await fixtureFn.call(context);
+  for (const fixtureFn of fixtureFns) {
+    try {
+      await fixtureFn.call(context);
+    } catch (err) {
+      console.error(err.stack);
```

Comment:
[Functionality] This is a good start, but I think we'll want to do more than just log the error to the console. I think we'll want to mark the test as failed too. It's easy to miss console logs that don't change the exit code.